core/mondoo-macos-security.mql.yaml

```diff
@@ -16,7 +16,7 @@ policies:
       desc: |-
         ## Overview
 
-        This policy provides prescriptive guidance for establishing a secure configuration posture for Apple macOS. This guide was tested against Apple macOS 10, 11, 12, and 13.
+        This policy provides prescriptive guidance for establishing a secure configuration posture for Apple macOS. This guide was tested against Apple macOS 10.15, 11, 12, 13, and 14.
```

Comment:
Given how long lived 10 was, we should remain specific about 10.15 and not earlier releases.

Reply:
You didn't test that policy on 10.1?